packages/config/src/ios/Updates.ts

```diff
@@ -83,3 +87,129 @@ export function setVersionsConfig(config: ExpoConfig, expoPlist: ExpoPlist) {
 
   return newExpoPlist;
 }
+
+function getBuildPhaseShellScriptPath(projectDir: string, config: ExpoConfig): string {
+  const buildScriptPath = projectHasModule(
+    'expo-updates/scripts/create-manifest-ios.sh',
+    projectDir,
+    config
+  );
+
+  if (!buildScriptPath) {
+    throw new Error(
+      "Could not find the build script for iOS. This could happen in case of outdated 'node_modules'. Run 'npm install' to make sure that it's up-to-date."
+    );
+  }
+
+  return path.relative(path.join(projectDir, 'ios'), buildScriptPath);
+}
+
+interface ShellScriptBuildPhase {
+  isa: 'PBXShellScriptBuildPhase';
+  name: string;
+  shellScript: string;
+  [key: string]: any;
+}
+
+function getBundleReactNativePhase(project: xcode.XcodeProject): ShellScriptBuildPhase {
+  const scriptBuildPhase = project.hash.project.objects.PBXShellScriptBuildPhase as Record<
+    string,
+    ShellScriptBuildPhase
+  >;
+  const bundleReactNative = Object.values(scriptBuildPhase).find(
+    buildPhase => buildPhase.name === '"Bundle React Native code and images"'
+  );
+
+  if (!bundleReactNative) {
+    throw new Error(`Couldn't find a build phase "Bundle React Native code and images"`);
+  }
+
+  return bundleReactNative;
+}
+
+export function setBuildPhaseShellScript(
+  projectDir: string,
+  config: ExpoConfig,
+  project: xcode.XcodeProject
+): xcode.XcodeProject {
```

Comment:
Why this function returns the `xcode.XcodeProject` type? It doesn't seem to be used anywhere.

Reply:
it should be used like this, I forgot to assign the returned value in cli part.

Technically speaking `xcode.XcodeProject` object is modified so returning is not strictly necessary